client/jetpack-cloud/sections/pricing/controller.tsx

```diff
@@ -17,7 +19,16 @@ export function jetpackPricingContext( context: PageJS.Context, next: () => void
 
 	context.store.dispatch( hideMasterbar() );
 	context.nav = <JetpackComMasterbar pathname={ lang ? path.replace( `/${ lang }`, '' ) : path } />;
-	context.header = <Header urlQueryArgs={ urlQueryArgs } />;
+	context.header = (
+		<Header
+			urlQueryArgs={ urlQueryArgs }
+			title={
+				isEnabled( 'jetpack/pricing-page-rework-v1' )
+					? translate( 'Best-in-class products for your WordPress site' )
+					: undefined
```

Comment:
This is good, though I suggest either put a comment like `//Default behavior inside component` or simply specify it here `translate( 'Security, performance, and marketing tools made for WordPress' )` to avoid confusion from other folks

Reply:
Yeah this make more sense.. I added the default string as suggested.. 👍